### ISSUE TITLE
R base version isn't pinned in environments

### ISSUE THREAD

**Issue by LMBradford** [R-ODAF/R-ODAF_Health_Canada#256]
I thought that R itself is pinned in the conda environments, but it is not. I guess whatever R version is installed on user's computers is what gets used. This will lead to problems if users have a newer R version that doesn't work well with our pinned older packages defined in the env yamls.

**Comment by mattjmeier** [comment]
Hmmm but the R interpreter that gets used is determined by conda right? I'm pretty sure the version of r for example on the gh action runner wasn't changing

**Comment by LMBradford** [comment]
You're right, it's determined by conda and it must get downloaded into the snakemake-created environments as a dependency of one (probably more) of the r packages we have in the yaml. The R version in my base environment is 4.4.1, but the version in my persistant R-DOAF_reports environment (downloaded a few months ago) is 4.2.3. I just installed the environment again and it's still version 4.2.3 . So maybe this isn't an issue at all.

**Comment by mattjmeier** [comment]
It might not be a bad idea to at least define the version, for reproducibility reasons though
